config/config.yaml

```diff
@@ -128,13 +131,13 @@ analysis:
         include: true
       # Create output files for each pathway that can be visualized with GraphSpace
       graphspace:
-        include: true
+        include: false
       # Create Cytoscape session file with all pathway graphs for each dataset
       cytoscape:
-        include: true
+        include: false
       # Machine learning analysis (e.g. clustering) of the pathway output files for each dataset
       ml:
-        include: true
```

Comment:
I merged a pull request that affects this part of the config file, which now created a merge conflict. I can resolve the conflict if you would like.